client/extensions/woocommerce/state/action-list/reducer.js

```diff
@@ -29,40 +25,30 @@ function handleActionListClear() {
 	return null;
 }
 
-function handleActionListStepStart( actionList, action ) {
-	const { stepIndex, time } = action;
-	const startTime = time || Date.now();
-	const step = actionList[ stepIndex ];
+function handleActionListAnnotate( actionList, action ) {
+	const { stepIndex, annotations } = action;
```

Comment:
You could do multi-level destructuring like `const { stepIndex, annotations: { startTime, endTime, error } } = action;` . Not everyone loves that, but I wanted to point it out :)